examples/all-clusters-app/all-clusters-common/src/bridged-actions-stub.cpp

```diff
@@ -129,3 +134,12 @@ Status ActionsDelegateImpl::HandleDisableActionWithDuration(uint16_t actionId, u
     // Not implemented
     return Status::NotFound;
 }
+
+void emberAfActionsClusterInitCallback(EndpointId endpoint)
+{
+    VerifyOrReturn(endpoint == 1);
```

Comment:
how do we know that ember is configured for endpoint 1 havint this cluster? We use "endpoint" below however this seems hard-coded.

Please add `static_assert` here on the endpoint config matching what the cluster expects.

Reply:
Added `emberAfContainsServer()` check.

Reply:
now if I add in ember a actions cluster on any other endpoint, code still compiles however the action cluster simply will not work. This will be painful to debug.

Reply:
Could we at least log in this case rather than just return?

Reply:
Added error log